simtools/Geppeto.yaml

```diff
@@ -1,24 +1,7 @@
 - name: Geppeto
 - website_url: ''
-- operating_system:
-  - windows: ''
-  - macOS: ''
-  - linux: ''
+- operating_system: ''
 - biological_level: ''
```

Comment:
This seems like a framework to me and not actually a simulator software. #1 #4 

Thoughts?

Reply:
It is indeed a framework, but it comes with a backend for simulation, so I think it could be listed. On the other hand, I am not sure whether it is ever supposed to be used by itself. Other tools like OpenSourceBrain build on top of it, if I am not mistaken.